docs/user/project_metadata.md

```diff
@@ -0,0 +1,88 @@
+# Projects metadata
+
+
+## Project-URL
+
+A package can contain optional [Project URL](https://packaging.python.org/en/latest/specifications/core-metadata/#project-url-multiple-use) 
+keys.
+
+PyPI renders these URLs on the package project page. They are also available in 
+the JSON data when requesting the package information.
+
+### Verified details
+
+![Verified details](assets/verified_details.png){ loading=lazy }
+
+
+[Trusted Publishing](trusted-publishers) allow PyPI to verify certain 
+information submitted by the package maintainer. Verified information appears in
```

Comment:
Let's be clear about what information we use for verification here -- username and repository name. We should also be clear about which providers we support this for and which we don't.

Reply:
I think this still needs to be resolved.